src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -248,6 +248,25 @@ std::pair<uint32_t, uint32_t> get_lora_dims_by_name(const std::string& state_nam
     return std::make_pair(low_rank_dim, full_rank_dim);
 }
 
+void copy_to_right(const ov::SoPtr<ov::ITensor>& src, const ov::SoPtr<ov::ITensor>& dst) {
+    OPENVINO_ASSERT(src->get_byte_size() <= dst->get_byte_size());
+    std::copy_n(reinterpret_cast<uint8_t*>(src->data()),
+                src->get_byte_size(),
+                reinterpret_cast<uint8_t*>(dst->data()) + dst->get_byte_size() - src->get_byte_size());
+}
+
+void fill_sliding_mask(const ov::SoPtr<ov::ITensor>& mask, int64_t curr_pos, int64_t window_size) {
+    auto start = curr_pos - window_size;
+    auto end = curr_pos + window_size;
```

Comment:
Could you please remove `window_size` here and add a comment about PyTorch behavior as we discussed?

Reply:
Could you please add a comment that `end` is inclusive because it reflects window size of past KV cache. 

P.S. Is it correct that we apply 1024 mask to the past context and apply new token (1025) in the current infer stage?

Reply:
I've excluded `end` and added a comment